earth2studio/data/weatherbench.py

```diff
@@ -0,0 +1,241 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pathlib
+import shutil
+from datetime import datetime
+
+import fsspec
+import gcsfs
+import numpy as np
+import xarray as xr
+import zarr
+from loguru import logger
+from modulus.distributed.manager import DistributedManager
+from tqdm import tqdm
+
+from earth2studio.data.utils import prep_data_inputs
+from earth2studio.lexicon import WBLexicon
+from earth2studio.utils.type import TimeArray, VariableArray
+
+LOCAL_CACHE = os.path.join(os.path.expanduser("~"), ".cache", "earth2studio")
+
+
+class WeatherBenchClimatology:
```

Comment:
Maybe update to WeatherBench2Climatology so it matches the repo.

Also could be WB2Climatology to shorten the name. :shrug:

Reply:
also maybe file name can be wb2.py to follow the current abbreviation trend